scylla-macros/src/from_row.rs

```diff
@@ -28,13 +29,19 @@ pub fn from_row_derive(tokens_input: TokenStream) -> TokenStream {
         }
     });
 
+    let fields_count = struct_fields.named.len();
     let generated = quote! {
         impl FromRow for #struct_name {
             fn from_row(row: scylla::frame::response::result::Row)
             -> Result<Self, scylla::cql_to_rust::FromRowError> {
                 use scylla::frame::response::result::CqlValue;
                 use scylla::cql_to_rust::{FromCqlVal, FromRow, FromRowError};
 
+                if #fields_count != row.columns.len() {
+                    return Err(FromRowError::WrongRowSize {
+                        expected: #fields_count, actual: row.columns.len()
```

Comment:
Nitpick: please put `expected` and `actual` field into separate lines (like `cargo fmt` would if it knew how to format macros).

Reply:
Done